noether_examples/launch/mesh_segmenter_server_client.launch

```diff
@@ -1,22 +1,13 @@
 <launch>
-    <arg name="filename"/>
-    <arg name="min_cluster_size" default="500"/>
-    <arg name="max_cluster_size" default="1000000"/>
-    <arg name="curvature_threshold" default="0.05"/>
-    <arg name="show_individually" default="false"/>
-    <arg name="save_outputs" default="false"/>
-
-
-    <node name="mesh_segmenter_server_node" type="segmentation_server" pkg="noether" output="screen" required="true" >
-      <!--Loads a particular test file: ply -->
-      <param name="filename" value="$(arg filename)" type="string"/>
-      <param name="min_cluster_size" value="$(arg min_cluster_size)" type="int" />
-      <param name="max_cluster_size" value="$(arg max_cluster_size)" type="int" />
-      <param name="curvature_threshold" value="$(arg curvature_threshold)" type="double" />
-      <param name="show_individually" value="$(arg show_individually)" type="bool" />
-      <param name="save_outputs" value="$(arg save_outputs)" type="bool" />
-    </node>
+  <arg name="filename"/>
+  <arg name="min_cluster_size" default="500"/>
+  <arg name="max_cluster_size" default="1000000"/>
+  <arg name="curvature_threshold" default="0.05"/>
+  <arg name="show_individually" default="false"/>
+  <arg name="save_outputs" default="false"/>
 
+  <!-- Launch the segmentation server -->
+  <node name="mesh_segmenter_server_node" type="segmentation_server" pkg="noether" output="screen" required="true" />
 
   <node name="mesh_segmenter_client_node" type="mesh_segmenter_client_node" pkg="noether_examples" output="screen" required="true" >
     <!--Loads a particular test file: ply -->
```

Comment:
I'm not sure why the mesh segmenter client and server are ran from the same launch file.  Wouldn't you just want to run the server only or is this a demo launch file?